File: C++ Solution/96a.cpp
#include <iostream>
#include <string>
using namespace std;
 
int main()
{
int count=0;
string str;
 
cin>>str;
 
for(int i=0;i<str.length();i++)
{
  if(str[i]==str[i+1])
  { 
   count++;
    if(count==6)
    {
     cout<<"YES"<<endl;
     return 0;
    }
  }
  else
  {  
   count=0;
  }
}
cout<<"NO"<<endl;
 
return 0;
}

File: Python solutions/CodeForces/96a.py
s = input().strip()
count = 0

for i in range(len(s) - 1):
    if s[i] == s[i + 1]:
        count += 1
        if count == 6:
            print("YES")
            exit()
    else:
        count = 0

print("NO")